include/deal.II/lac/trilinos_tpetra_vector.h

```diff
@@ -739,6 +787,52 @@ namespace LinearAlgebra
        */
       DeclException0(ExcVectorTypeNotCompatible);
 
+
+      /**
+       * Attempt to write into an off-processor entry, but either the vector is
+       * compressed and can not be automatically decompressed, or no locally
+       * relevant index set was provided to the vector.
+       *
+       * @ingroup Exceptions
+       */
+      DeclExceptionMsg(ExcAccessOffProcessorEntry,
+                       "Attempted to access a vector entry that does not "
```

Comment:
Let's write this in the present tense:
```suggestion
                       "You are attempting to access a vector entry that does not "
```